### ISSUE TITLE
Discrepancy reported and reproduced results of linear eval

### ISSUE THREAD

**Issue by WannesVanwinsen** [TengdaHan/CoCLR#54]
Good afternoon,

`python main_classifier.py --net s3d --dataset ucf101 --seq_len 32 --ds 1 --batch_size 32 --train_what last --epochs 100 --schedule 60 80 --optim sgd --lr 1e-1 --wd 1e-3  --pretrain ../feature/CoCLR-k400-rgb-128-s3d.pth.tar`

After running the linear evaluation on UCF101 using the suggested parameters I obtained a final top-1 accuracy of 66.2 rather than the expected 74.5 reported in Table 2 row 9 of your paper. I saw that you expect a margin of 1-2% difference in these results, which is much less than what I am seeing myself. Are the hyperparameters that are provided in the repository the exact same as those used to obtain the results given in your Table 2? I am running the experiments using a single T4 GPU. Thank you in advance!
